### PR TITLE
Added updates to portfolio

### PR DESCRIPTION
**Pull request by direakanbi**
# My Microverse Portfolio Project
 This projected was actualised for the creation of my portfolio.

[](https://postimg.cc/t1P2v3RT)
[](https://postimg.cc/Fk56rhG9)
[](https://postimg.cc/LhL7yZ6h)

# Additional description about the project and its features.

# Built With
  1. HTML
  2. CSS
  3. Linters (Webhint, Stylelint, Lighthouse).

# Getting Started

To get a local copy up and running follow these simple example steps.

Prerequisites
Setup: Clone direakanbi/microverse-portfolio1 into the destination folder of your local machine.
Install: Install linters with the npx commands given in the linters.yml file.
Usage: N/A
Run tests: Run syntax tests with the use of the npx commands given in the linters.yml file.
Deployment: Run index.html on your local server.

# Authors

GitHub: @direakanbi
Twitter: @direakanbi
LinkedIn: Akanbi Dire

Contributions, issues, and feature requests are welcome!

Feel free to check the issues page.

Show your support
Give a ⭐️ if you like this project!

📝 License
This project is MIT licensed

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @direakanbi,

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️

- [ ] The PR description should only contain the features implemented and not the contents of the readme file. Write a short description of all the implemented features.

_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -36,6 +36,186 @@ <h1>GLAD TO SEE <br>YOU!</h1>
...
+            <li><a href="#" aria-label="twitter"><i class="fab fa-twitter"></i></a></li> 
+            <li><a href="#" aria-label="linkedin"><i class="fab fa-linkedin-in"></i></a></li>
+            <li><a href="#" aria-label="medium"><i class="fab fa-medium-m"></i></a></li>
+            <li><a href="#" aria-label="gtihub"><i class="fab fa-github"></i></a></li> 
+        </ul>
+        </div>
+    
+        <button class="see">Get My Resume</button>
+    </li>
+
```
Comment (mke2111) [inline index.html:174]: - [ ] Add the necessary styles to the about section. Compare your page with the original
<div class="flex">


</div>

- Focus on the background styles and color for the entire about section including the languages section.
- Add the border radius to the about section background.

File: index.html [hunk 2]
```diff
@@ -36,6 +36,186 @@ <h1>GLAD TO SEE <br>YOU!</h1>
             <li><a href="#"><i class="fab fa-github"></i></a></li> 
         </ul>
       </div>
```
Comment (mke2111) [inline index.html:38]: OPTIONAL

- [ ] When you implemented the projects and about sections, bug were introduced that affected the design of the section above. You may need to give it a fix.
Screenshot from 2021-09-20 14-41-53
.

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashbaord. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**